devolutions-session/src/dvc/fs.rs

```diff
@@ -1,15 +1,11 @@
 use std::path::PathBuf;
 
-use windows::core::PCWSTR;
-use windows::Win32::Storage::FileSystem::{MoveFileExW, MOVEFILE_DELAY_UNTIL_REBOOT};
-
-use win_api_wrappers::utils::WideString;
-
 /// Guard for created temporary file. Associated file is deleted on drop.
 pub struct TmpFileGuard(PathBuf);
```

Comment:
thought: I’m fairly sure we already discussed about that, but did you consider using or integrating with the tempfile crate? (https://crates.io/crates/tempfile)
I think it would be worth to mention this crate in a module doc comment, and explain why we are not integrating with it

Reply:
this type is actually is using `tempfile` under the hood (see `new()`)

Reply:
Oh, right! :joy: